<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rdsIconWindow</class>
 <widget class="QDialog" name="rdsIconWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>94</width>
    <height>27</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Yarra RDS</string>
  </property>
  <widget class="QLabel" name="staticIconLabel">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>4</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="rds.qrc">:/images/topIcon.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="animIconLabel">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>3</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="rds.qrc">:/images/topIcon_run.gif</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="errorIconLabel">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>4</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="rds.qrc">:/images/topIcon_error.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="rds.qrc"/>
 </resources>
 <connections/>
</ui>
